src/components/PackageDocs/Body.tsx

```diff
@@ -0,0 +1,68 @@
+import { LinkIcon } from "@chakra-ui/icons";
+import { Heading, As } from "@chakra-ui/react";
+import ChakraUIRenderer from "chakra-ui-markdown-renderer";
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import { sanitize } from "../../util/sanitize-anchor";
+
+type HeadingResolverProps = {
+  level: number;
+  children: React.ReactNode;
+};
+
+function Headings({ level, children }: HeadingResolverProps) {
+  const size: string = ["2xl", "xl", "lg", "md", "sm", "xs"][level - 1];
+  const elem = `h${level}` as As<any>;
+
+  // Use DOMParser to look for data attribute for link ID
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(
+    React.Children.toArray(children).join(""),
+    "text/html"
+  );
+
+  const dataElement = doc.querySelector(
+    "span[data-heading-title][data-heading-id]"
+  ) as HTMLElement;
+  const title =
+    dataElement?.dataset.headingTitle ??
+    React.Children.toArray(children)
+      .reduce((accum: string, child): string => {
+        if (typeof child === "string") {
+          return `${accum}${child}`;
+        }
+        return accum;
+      }, "")
+      .trim();
+  const id = dataElement?.dataset.headingId ?? sanitize(title);
```

Comment:
Are you sure this is working properly? I'm still seeing titles as anchor ids, not `FQN`s:

![Screen Shot 2021-06-07 at 12 34 03 PM](https://user-images.githubusercontent.com/1428812/120994665-5e385b80-c78d-11eb-819e-b8e381d3f51e.png)

Also - are the defaults being applied here to handle the user defined README? Would be good to document them and ideally merge to a single branch.

Reply:
Fixed this, it wasn't parsing the `span` elements generated from the markdown render correctly. 😄